source/MBF.SAMD51.GPIO.pas

```diff
@@ -42,14 +42,37 @@ interface
   Pad2=$120000;
   Pad3=$130000;
 
+  PINMODE_INPUT           = 0;
+  PINMODE_OUTPUT          = 1;
+  PINMODE_INPUT_PULLUP    = 2;
+  PINMODE_INPUT_PULLDOWN  = 3;
+
```

Comment:
You are defining things that should be defined in TPinMode, reason is to enable Lazarus to better help you by offering the choices available and to make the code more typesafe.

Reply:
These four defines are meant to be Wio Terminal fatfreertos only. The are only used inside the more general GPIO functions. I will more them towards the proper loacation. They are not meant to be visible.

Reply:
https://github.com/LongDirtyAnimAlf/mbf-freertos/commit/4d4ae0c598f97e3be505fe03dffc506b982e9294